docs/content/tutorials/tutorial-dapr/snippets/dapr.bicep

```diff
@@ -59,6 +59,8 @@ resource frontend 'Applications.Core/containers@2023-10-01-preview' = {
       env: {
         // An environment variable to tell the frontend container where to find the backend
         CONNECTION_BACKEND_APPID: 'backend'
+        // An environment variable to override the default port that .NET Core listens on
+        ASPNETCORE_URLS: 'http://*:8080'
```

Comment:
Does it only serve through HTTP?

Reply:
@ytimocin -- not sure, but http is the relevant protocol for our tutorial, so I think this should be okay?

Reply:
We only need HTTP for the tutorial. .NET does both (even in development scenarios).